src/core/public/ui_settings/ui_settings_client.ts

```diff
@@ -141,7 +141,7 @@ You can use \`IUiSettingsClient.get("${key}", defaultValue)\`, which will just r
   }
 
   private validateScope(key: string, scope: UiSettingScope) {
-    const definition = this.cache[key];
+    const definition = this.cache[key] || {};
```

Comment:
Could we add a logging to warn the user in this case?

Reply:
Good idea.

Reply:
> Nit: could we add a logging to warn the user in this case?

Why we need to add warning for this case?